### ISSUE TITLE
Redux Devtools not working for me

### ISSUE THREAD

**Issue by rjmunro** [chentsulin/electron-react-boilerplate#262]
When I try to use the redux panel in the dev tools, I just get a blank page, and this appears in the terminal where I ran `npm run dev`: console:

    [1936:0701/143626:ERROR:CONSOLE(22253)] "Uncaught TypeError: Cannot read property 'get' of undefined", source: chrome-extension://redux-devtools/js/devpanel.bundle.js

Possibly related: there is a git submodule in tools/react-devtools/ but when I do `git submodule init`, I get:

    fatal: No url found for submodule path 'tools/react-devtools' in .gitmodules

But react-devtools are working. It's redux that aren't.

**Comment by jhen0409** [comment]
You need upgrade the extension to new version:

* Call `BrowserWindow.removeDevToolsExtension('Redux DevTools')`
* Currently you must to delete extension folder, please wait https://github.com/GPMDP/electron-devtools-installer/pull/6 for forceDownload feature back.
  * macOS: `/Users/<user>/Library/Application\ Support/Electron/extensions`
  * You can get path with call `app.getPath('userData')`

**Comment by jhen0409** [comment]
The `tools/react-devtools` is unnecessary for now, we should remove it..

**Comment by jhen0409** [comment]
```
[1936:0701/143626:ERROR:CONSOLE(22253)] "Uncaught TypeError: Cannot read property 'get' of undefined", source: chrome-extension://redux-devtools/js/devpanel.bundle.js
```
This is from old redux-devtools-extension version (2.2.0), it should be fixed in v2.2.1.

Now we can use `UPGRADE_EXTENSIONS=1 npm run start-hot` upgrade extensions (#263), please update dependencies first.

**Comment by rjmunro** [comment]
That seems to have fixed things - thanks.
